### ISSUE TITLE
Not working after I add distribution plugin(ios)

### ISSUE THREAD

**Issue by vforv** [EddyVerbruggen/nativescript-plugin-firebase#1178]
There is a bug in plugin, it works fine when I add development account and create new app id. After I add production account it stops working even for develmpment.

I start gettings this error message:

`{"multicast_id":8041127734126337384,"success":0,"failure":1,"canonical_ids":0,"results":[{"error":"NotRegistered"}]}`

**Comment by EddyVerbruggen** [comment]
Did you adjust the apn environment in the entitlements file perhaps?

**Comment by vforv** [comment]
Yes, go trought each step minimum 10 times... Now I installed TestFlight on another iPhone device and added Keys to firebase and it works on that device.
Very wierd...

**Comment by curtiscarlson** [comment]
Did you ever solve this.  I am on day 5 of deleting keys and certificates, reinstalling, rebuilding putting on testflight, log into app, send test notification, and it fails.  It worked in development and for my life it will not work in production.  I am furstrated to tears here.
